pkg/cmd/server/api/types.go

```diff
@@ -980,3 +983,19 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// PodLimitRequestConfig is the configuration for the PodLimitRequest
+// admission controller which overrides user-provided container request/limit values.
+type PodLimitRequestConfig struct {
```

Comment:
I'm not enamored of the name either, just didn't have anything else come to mind. Something that expresses that the purpose is cluster-wide control of resource allocation, though this is just about pods at the moment. Frankly "RequestLimit"-anything probably doesn't lead the mind in the right direction. How about "ClusterResourceOverrides"?